mrmustard/lab_dev/states.py

```diff
@@ -70,5 +70,5 @@ def representation(self) -> Bargmann:
         num_modes = len(self.modes)
         A = math.zeros(shape=(num_modes, num_modes), dtype=math.complex128)
         B = math.zeros(shape=(num_modes), dtype=math.complex128)
-        C = 1.0
+        C = math.cast(1.0, dtype=math.complex128)
```

Comment:
Or change it as you suggested:
```suggestion
        C = 1.0 + 0j
```